src/cmd/linuxkit/run.go

```diff
@@ -20,7 +20,8 @@ func runUsage() {
 	fmt.Printf("  aws\n")
 	fmt.Printf("  azure\n")
 	fmt.Printf("  gcp\n")
-	fmt.Printf("  hyperkit [macOS]\n")
+	fmt.Printf("  virtualization [macOS]\n")
```

Comment:
this is a terrible name - not your fault - as a lot of these other options are also `virtualization`
not sure if "vf" is any better. or `virt.framework` or maybe just `virtualization.framework`.
since it's a default option i hope no-one will ever have to type it in full :laughing: